.github/workflows/build_deploy.yml

```diff
@@ -22,8 +22,12 @@ jobs:
           python -m pip install --upgrade pip
           pip install -q -r requirements.txt -c constraints.txt
 
-      - name: Run tests with make test
-        run: make testlight
+      - name: Run spellcheck
+        run: make spellcheck
```

Comment:
make test

+1 for switching back to mandatory linkcheck. 
Sorry for the noise, I am sometimes somehow impatiant. Linkcheck is important.

„Zwei Seelen wohnen, ach! in meiner Brust, / Die eine will sich von der andern trennen.“ — Johann Wolfgang von Goethe
“Two souls live, alas! in my chest / One wants to separate from the other. ”- Johann Wolfgang von Goethe